gpii/configs/gpii.config.untrusted.development.local.dr.production.json5

```diff
@@ -0,0 +1,52 @@
+/**
+ * This configuration runs the entire system locally under the hybrid Flow Manager architecture and in development mode.
+ *
+ * It uses a multi-server Kettle configuration to run the Untrusted Flow Manager, the dynamic Device Reporter and a local version of the Cloud Based Flow Manager within the same Kettle process, but on different ports.
+ *
+ * * The Untrusted Flow Manager runs on port 8081
+ * * The Cloud Based Flow Manager runs on port 8084
+ *
+ * * The following components are running on the local machine:
+ * ** Untrusted Flow Manager
+ * *** dynamic deviceReporter
+ * *** flowManager (untrusted, browser channel)
+ *
+ * ** Cloud Based Flow Manager
+ * *** authServer
+ * *** canopyMatchMaker
+ * *** flowManager (cloud based oauth2)
+ * *** preferencesServer
+ * *** solutionsRegistry
+ */
+{
+    "type": "gpii.config.untrusted.development.local.dr.production",
+    "options": {
+        "gradeNames": ["kettle.multiConfig.config"],
+        "configs": {
+            "localConfig": {
+                "configName": "gpii.config.untrusted.development.dr.production",
```

Comment:
It seems this line is the only difference between this config and [`gpii.config.untrusted.development.local.json5`](https://github.com/GPII/universal/blob/master/gpii/configs/gpii.config.untrusted.development.local.json5). To simplify, can you use `mergeConfigs` block to inherit `gpii.config.untrusted.development.local.json5` then inject this line with `distributeOptions`?

This comment also applies to other files.